.travis.yml

```diff
@@ -15,10 +16,16 @@ env:
     SIMPLETEST=0
 
 php:
-  - '5.6'
+  - 7.0
+  - 7.1
+  - 7.2
+  - nightly
 
-node_js:
-  - '4'
+matrix:
+  fast_finish: true
+  allow_failures:
```

Comment:
Should we test for 7.2 and nightly if we allow failures? Doesn't the job take too long then? Does the fast_finish setting makes the job successful even if it has to run on 7.2 and nightly?